### ISSUE TITLE
Built-In yamlfmt Lint Command Uses Format-Oriented Arguments

### ISSUE THREAD

**Issue by mattiasgronlund** [rvben/rumdl#595]
### rumdl version

rumdl 0.1.84

### Installation method

cargo install

### Bug description

The built-in `yamlfmt` code-block tool definition uses command arguments that do not match yamlfmt's lint-from-stdin mode. As a result, YAML code-block linting can invoke yamlfmt with an invalid or format-oriented argument shape instead of validating stdin with `yamlfmt -lint -`.

### Steps to reproduce

1. Install `yamlfmt`.
2. Create `.rumdl.toml`:

   ```toml
   [code-block-tools]
   enabled = true

   [code-block-tools.languages.yaml]
   lint = ["yamlfmt"]
   ```

3. Create `test.md`:

   ````markdown
   ```yaml
   key: value
   ```
   ````

4. Run:

   ```bash
   rumdl check test.md
   ```

### Expected behavior

rumdl invokes the built-in yamlfmt linter as:

```bash
yamlfmt -lint -
```

### Actual behavior

The built-in definition builds an argument vector based on `yamlfmt -` with `-dry` as lint args, which does not represent the intended lint-from-stdin command.

### Configuration

```toml
See reproduction config above.
```

### Operating system

Linux

**Comment by rvben** [comment]
Thanks @mattiasgronlund — both for the report and the fix in #606!
Will be in the next release.
